### PR TITLE
Updates to the Annual Report Process and Template for LFDT

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Comment by swcurran** [comment]
Changes in the PR:

- Hyperledger references to LFDT
- lifecycle "stage" references --> "status"
- Change references to Maintainers to TSC
- Added "TBD Link" for links that are TBD (e.g., Discord channel)
- Fixed Markdown lint warnings
- General text cleanup

**Review by petermetz — APPROVED** [review]

File: project-reports/0000-annual-review-template.md [hunk 1]
```diff
@@ -6,36 +6,43 @@ grand_parent: LF Decentralized Trust TAC
...
+<mark>_Copy this template to the subdirectory for the current year and name the file `YYYY-annual-Project-Name.md` (e.g., `2024-annual-lf-decentralized-trust-amazing.md`). Update the information above to change the `title` to the `YYYY Annual Review Project Name` (e.g., `2024 Annual Review LF Decentralized Trust Amazing`), the `parent` to `YYYY` (e.g., 2024), the `grand_parent` to `Project Updates`, and remove the `nav_exclude` line. Text between `<mark></mark>` are instructions. Please remove when section has been completed._
 </mark>
 
-# Project Health
-<mark>_Include a link to your project’s [LFX Insights page](https://insights-v2.lfx.linuxfoundation.org/projects). We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add color to the numbers and graphs we will see on Insights._
+## Project Health
```
Comment (arsulegai) [inline project-reports/0000-annual-review-template.md:12]: [nitpick] I’m curious why the entire file appears as different. It seems like an extra # was added for the headers.

File: governing-documents/project-annual-review.md [hunk 2]
```diff
@@ -10,81 +10,89 @@ nav_order: 5
...
-- Creating the annual review by filing a PR in GitHub under [hyperledger/toc/project-reports/](https://github.com/hyperledger/toc/tree/gh-pages/project-reports)
-- Sending email to the [TAC mailing list](mailto:toc@lists.hyperledger.org) so that the community knows the PR is there and can comment on it
-- Optionally attending the public TAC meeting to engage in TAC-led discussion around the project. The project's maintainers are invited to use this time to voice any concerns and requests for help they may have that are not captured in the PR (or highlight asks within the PR)
+### Project TSC
+
+- Agreeing between themselves who will complete the annual review.
+- Informing the community about the annual review and requesting input.
+- Creating the annual review by filing a PR in GitHub under [lf-decentralized-trust/toc/project-reports/](https://github.com/lf-decentralized-trust/toc/tree/gh-pages/project-reports).
+- Sending email to the [TAC mailing list](mailto:toc@lists.lf-decentralized-trust.org) and to the project community so that the community knows the PR is there and can comment on it.
```
Comment (arsulegai) [inline governing-documents/project-annual-review.md:73]: Q: Is the mailing list a TODO?

File: governing-documents/project-annual-review.md [hunk 3]
```diff
@@ -10,81 +10,89 @@ nav_order: 5
...
-### Project Maintainers
-- Agreeing between themselves who will complete the annual review and informing the TAC of who will be completing the report
-- Creating the annual review by filing a PR in GitHub under [hyperledger/toc/project-reports/](https://github.com/hyperledger/toc/tree/gh-pages/project-reports)
-- Sending email to the [TAC mailing list](mailto:toc@lists.hyperledger.org) so that the community knows the PR is there and can comment on it
-- Optionally attending the public TAC meeting to engage in TAC-led discussion around the project. The project's maintainers are invited to use this time to voice any concerns and requests for help they may have that are not captured in the PR (or highlight asks within the PR)
+### Project TSC
+
+- Agreeing between themselves who will complete the annual review.
+- Informing the community about the annual review and requesting input.
+- Creating the annual review by filing a PR in GitHub under [lf-decentralized-trust/toc/project-reports/](https://github.com/lf-decentralized-trust/toc/tree/gh-pages/project-reports).
```
Comment (arsulegai) [inline governing-documents/project-annual-review.md:72]: Q: Is the github link a TODO?

File: governing-documents/project-annual-review.md [hunk 4]
```diff
@@ -10,81 +10,89 @@ nav_order: 5
...
+- Documenting their findings within a thread in the TAC channel (TBD `#tac`) for discussion. The thread should contain:
+    1. Important facts about the project that could influence the TAC's decision around the future of the project, its current status, and paths to other statuses.
+    2. Whether the project's view of themselves is accurate and the ask of the TAC is reasonable to assist the project moving forward.
+- Lead the annual review discussion at the TAC meeting, providing a summary of the project by leveraging the private thread's content as the basis of discussion. The discussion typically focuses on what is going well with the project and areas to improve.
+- After the public meeting wraps up, summarizing the discussion on the PR in the form of a comment to document information for the project and community.
 
 ### TAC Secondary Member
-- Working closely with the lead TAC member to review and corroborate their findings
+
+- Working closely with the lead TAC member to review thew project's annual report, and corroborate and publish their findings.
```
Comment (arsulegai) [inline governing-documents/project-annual-review.md:88]:
```suggestion
- Working closely with the lead TAC member to review thew project's annual report, and to corroborate and publish their findings.
```

File: governing-documents/project-annual-review.md [hunk 5]
```diff
@@ -10,81 +10,89 @@ nav_order: 5
...
+- Send an email to the **LINK TBD** `TAC mailing list` so that the community knows the PR is there and can comment on it.
 
 ## Potential Outcomes
+
 The outcome of the annual review is:
 
-* A majority of the TAC members agree to continue the project at its current stage, or
-* A majority of the TAC members recommend that a project be moved to a new stage, including `End of Life`, or
-* If the TAC cannot come to a consensus, the project will remain at its current stage.
+- A majority of the TAC members agree to continue the project at its current status, or
```
Comment (arsulegai) [inline governing-documents/project-annual-review.md:55]: Q: Is the term `project's status` broader than `project's stage` or do they carry the same meaning? If `status` is broader, does it imply an inferred condition based on the project's report and current stage?

File: governing-documents/project-annual-review.md [hunk 6]
```diff
@@ -10,81 +10,89 @@ nav_order: 5
...
-* The PR should include a file called `./<year>/<year>-annual-<Project-Name>.md` (e.g., `2024/2024-annual-Hyperledger-Amazing.md`) using [the annual review template](../project-reports/0000-annual-review-template.md)
-    - Update the information at the top of the file:
-        - change the `title` line to `YYY Annual Review Project Name` (e.g., `title: 2024 Annual Review Hyperledger Amazing`)
-        - change the `parent` line to `YYYY` (e.g., `parent: 2024`)
-        - change the `grand_parent` to `Project Updates` (i.e., `grand_parent: Project Updates`)
-        - remove the `nav_exclude` line
-    - Text between `<mark></mark>` are instructions. Please remove when section has been completed.
-* Send an email to the [TAC mailing list](mailto:toc@lists.hyperledger.org) so that the community knows the PR is there and can comment on it.
+Project TSC members are responsible for agreeing between them who will complete the annual review. One of the TSC members should create the review in GitHub under [lf-decentralized-trust/toc/project-reports/](https://github.com/lf-decentralized-trust/toc/tree/gh-pages/project-reports).
```
Comment (arsulegai) [inline governing-documents/project-annual-review.md:39]:
```suggestion
Project TSC members are responsible for agreeing between them who will complete the annual review. One of the project's TSC members should create the review in GitHub under [lf-decentralized-trust/toc/project-reports/](https://github.com/lf-decentralized-trust/toc/tree/gh-pages/project-reports).
```

Comment (arsulegai) [inline governing-documents/project-annual-review.md:39]: Q: Should the github link be a TODO?

File: governing-documents/project-annual-review.md [hunk 7]
```diff
@@ -10,81 +10,89 @@ nav_order: 5
...
+**NOTE:** If a project has genuinely stalled the project's TSC can save everyone’s time and effort by archiving the project and moving it to `End of Life`.
 
 ## Process
+
 The process for annual reviews is as follows:
 
-* A TAC representative will be chosen on a round-robin basis to lead the review once the project files a PR. This representative should be someone that is not directly involved with the project.
-* A secondary representative will be chosen on a round-robin basis to observe and corroborate the findings of the lead. This representative should be someone that is not directly involved with the project.
-* The lead TAC member reviews the content of the PR and analyzes the project for community health indicators, their findings are placed within a thread in the TAC channel ([#toc](https://discord.com/channels/905194001349627914/941384040316018790)) for discussion. The thread should contain:
-    1. important facts about the project that could influence the TACs decision around the future of the project, its current stage, and path to other stages.
```
Comment (arsulegai) [inline governing-documents/project-annual-review.md]: I notice that these sentences have been removed. It seems necessary to establish guidance that suggests the review should be objective and holistic, considering all aspects such as encouraging projects to plan for the next stage in their lifecycle, as well as ensuring ongoing viability.

File: governing-documents/project-annual-review.md [hunk 8]
```diff
@@ -10,81 +10,89 @@ nav_order: 5
...
 
 ## Timeline
-Annual reviews will replace the Q1 quarterly report.  **If your annual review is not submitted within two months of notification, we will take this as a sign that the project is not under active maintenance and the TAC is likely to decide to archive the project and move it to End of Life status.**
 
-**NOTE:** If a project has genuinely stalled we can save everyone’s time and effort by archiving the project and moving it to `End of Life`.
+Annual reviews will replace the Q1 quarterly report.  **If a project's annual review is not submitted within two months of notification, we will take this as a sign that the project is not under active maintenance and the TAC is likely to decide to archive the project and move it to End of Life status.**
```
Comment (arsulegai) [inline governing-documents/project-annual-review.md:18]:
```suggestion
Annual reviews will replace the Q1 quarterly report.  **If a project's annual review is not submitted within two months of notification, the TAC will take this as a sign that the project is not under active maintenance and is likely to decide to archive the project and move it to End of Life status.**
```

File: governing-documents/project-annual-review.md [hunk 9]
```diff
@@ -10,81 +10,89 @@ nav_order: 5
...
+
+**NOTE:** If a project has genuinely stalled the project's TSC can save everyone’s time and effort by archiving the project and moving it to `End of Life`.
```
Comment (arsulegai) [inline governing-documents/project-annual-review.md:20]: Q: Would project's TSC decide to EOL their project or do they bring it to LFDT TAC for the move?

File: project-reports/0000-annual-review-template.md (hunk 1 above)
Comment (swcurran) [reply project-reports/0000-annual-review-template.md:12]: That was cleanup of the headers — fixing Markdown Lint warnings.  The front matter puts in the “#” header, so the rest should be “##” headers.

File: governing-documents/project-annual-review.md (hunk 9 above)
Comment (swcurran) [reply governing-documents/project-annual-review.md:20]: I believe the TSC has the governance authority to do it. The mechanics would be situational. This sentence is a cleanup / clarification (at least it clarified it for me) on what was in the document already.

File: governing-documents/project-annual-review.md (hunk 5 above)
Comment (swcurran) [reply governing-documents/project-annual-review.md:55]: As per the discussion at the TOC call on 20240829 — the term “status” and “stage” were used seemingly interchangeably in the document.  I didn’t think that was clear, so I checked the “lifecycle” document and it uses the term “status”, so I changed all “stage” references to “status”.

File: governing-documents/project-annual-review.md (hunk 3 above)
Comment (swcurran) [reply governing-documents/project-annual-review.md:72]: I think I just need to change `toc` to `tac` in it.  But a link check should be run on all of these docs after publication to make sure they are right.

File: governing-documents/project-annual-review.md (hunk 2 above)
Comment (swcurran) [reply governing-documents/project-annual-review.md:73]: Definitely need a `toc` to `tac`, but yes, it is a ToDo.

File: governing-documents/project-annual-review.md (hunk 8 above)
Comment (swcurran) [reply governing-documents/project-annual-review.md:18]: Fixed

File: governing-documents/project-annual-review.md (hunk 6 above)
Comment (swcurran) [reply governing-documents/project-annual-review.md:39]: Fixed

Comment (swcurran) [reply governing-documents/project-annual-review.md:39]: The links can be checked by a link checker when the new repo is used.  I don’t think we need a todo on each where we probably have it right.  I did put a couple of “TBDs” where I didn’t know at all what the link would be.

File: governing-documents/project-annual-review.md (hunk 3 above)
Comment (swcurran) [reply governing-documents/project-annual-review.md:72]: Changed the `toc` to `tac` where appropriate. There is one instance that was purposely left.

File: governing-documents/project-annual-review.md (hunk 2 above)
Comment (swcurran) [reply governing-documents/project-annual-review.md:73]: Just made the `tac` to `toc` change, but I think a link checker can find this one if it is wrong.

File: governing-documents/project-annual-review.md (hunk 4 above)
Comment (swcurran) [reply governing-documents/project-annual-review.md:88]: Fixed

**Comment by swcurran** [comment]
I obviously have way too many commits in this — branched from the wrong fork.  Will close this, open a new PR and reference this  PR to capture the review comments made.  I’ve addressed all of the comments from the review.

**Pull request closed without merging**